app.js

```diff
@@ -14,6 +15,12 @@ app.use(mount('/', require('route/index').routes()))
 app.use(mount('/', require('route/user').routes()))
 app.use(mount('/admin', require('route/superadmin').routes())) // Super-admin API endpoints
 
+if (process.env.SERVE_DOCS) {
```

Comment:
```suggestion
if (/^(true|yes|on)$/i.test(process.env.SERVE_DOCS)) {
```

Reply:
I'll just do `JSON.parse(process.env.SERVE_DOCS)`. It should be a boolean anyway.